crates/pink/runtime/src/capi/ecall_impl.rs

```diff
@@ -243,3 +245,13 @@ impl ecall::ECalls for ECallImpl {
         phala_git_revision::git_revision().to_string()
     }
 }
+
+/// Clip gas limit to 1 second for tx, 10 seconds for query
```

Comment:
1 sec for tx is even too high. Consider reduce to 500ms maybe?

Reply:
Sounds OK